html/modules/custom/hr_paragraphs/hr_paragraphs.module

```diff
@@ -1954,6 +1954,12 @@ function hr_paragraphs_form_alter(&$form, $form_state, $form_id) {
         $form['langcode']['#access'] = FALSE;
       }
     }
+
+    // Restrict access to status.
+    $user = \Drupal::currentUser();
```

Comment:
Is it problematic that the previous `if()` block also defines `$user` inside the conditional?

Reply:
No, all is statically cached